src/jdk.jdwp.agent/windows/native/libjdwp/util_md.h

```diff
@@ -29,16 +29,11 @@
 #include <stddef.h>      /* for uintptr_t */
 #include <stdlib.h>      /* for _MAx_PATH */
 
-typedef unsigned __int64 UNSIGNED_JLONG;
+typedef unsigned long long UNSIGNED_JLONG;
```

Comment:
This change has nothing to do with _sprintf.  Not sure why it's being made here.

Reply:
It was a small change, so I thought I could make it out of convenience. I'll switch it out to a separate changeset

Reply:
There are lots of uses of those type names.  If they are going to be cleaned up, I'd prefer that get done as
as separate task, so I don't need to think about whether it's okay to do so while in the context of this change.
I don't know if __int64 == long long (probably is), but I'm pretty sure I remember seeing a comment on some
use of __int32 suggesting it was not the same as what someone expected.